jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -590,8 +603,24 @@ public JaegerTracer build() {
             .withMetrics(metrics)
             .build();
       }
+      return createTracer(serviceName, reporter, sampler, registry, clock, metrics, tags,
+          zipkinSharedRpcSpan, scopeManager, baggageRestrictionManager, expandExceptionLogs, objectFactory);
+    }
+
+    protected JaegerTracer createTracer(String serviceName,
```

Comment:
We used to have an auto-formatted in this repository, not sure it's still in effect. But pretty sure this style is ugly as hell - either inline them, or start from a new line so that they are only slightly indented (the way go fmt does)

Reply:
Lol same explanation as above.